tests/test_infer/test_kvcache_manager.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+import pytest
+import torch
+
+from colossalai.logging import disable_existing_loggers
+from colossalai.shardformer.inference import MemoryManager
+from colossalai.testing import rerun_if_address_is_in_use, spawn
+
+BATCH_SIZE = 4
+INPUT_LEN = 16
+OUTPUT_LEN = 8
+LAYER_NUM = 4
+HEAD_NUM = 32
+HEAD_DIM = 128
+
+
+def create_cache_manager(rank, world_size, port, batch_size, input_len, output_len, layer_num, head_num, head_dim):
+    os.environ['RANK'] = str(rank)
+    os.environ['LOCAL_RANK'] = str(rank)
```

Comment:
Why the `rank` and `local rank` are same?

Reply:
dumbly set the same for testing